cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/DistributedProgramRunner.java

```diff
@@ -604,7 +599,18 @@ private ProgramOptions updateProgramOptions(ProgramOptions options,
    * classpath.
    */
   @Nullable
-  private URI getLogBackURI(Program program) throws URISyntaxException {
+  private URI getLogBackURI(Program program, Arguments args, File tempDir) throws URISyntaxException, IOException {
```

Comment:
Not directly related to these changes - do you if it's sufficient to handle logback.xml, or do we also care about logback-container.xml (see L624 below)?

Reply:
Based on [the docs](https://cdap.atlassian.net/wiki/spaces/DOCS/pages/480346897/Application+Logback), it sounds like `logback-container.xml` is the default configuration and `logback.xml` should be used when it exists